tests/unit_tests/Commands/Self/ConsoleCommandTest.php

```diff
@@ -28,6 +30,30 @@ public function setUp()
      */
     public function testConsole()
     {
+        if (version_compare(PHP_VERSION, '7.1.0') < 0) {
```

Comment:
Suggestion: use `$this->markTestIncomplete("Console does not work on php7.1.");` when PHP >= 7.1.0.